common-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala

```diff
@@ -7,17 +7,17 @@ import com.socrata.datacoordinator.id.{ColumnId, UserColumnId}
 import com.socrata.datacoordinator.truth.loader.sql.AbstractRepBasedDataSqlizer
 import com.socrata.datacoordinator.truth.sql.SqlColumnRep
 import com.socrata.datacoordinator.util.CloseableIterator
+import com.socrata.pg.query.QueryResult.QueryRuntimeError
```

Comment:
I see some new imports but not any new code in this file

Reply:
QueryRuntimeError isn't new.  The order of imports was changed.